e2e/staticsite/staticsite_test.go

```diff
@@ -0,0 +1,130 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package staticsite_test
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/aws/copilot-cli/e2e/internal/client"
+)
+
+var _ = Describe("Static Site", func() {
+	Context("when creating a new app", Ordered, func() {
+		var appInitErr error
+
+		BeforeAll(func() {
+			_, appInitErr = cli.AppInit(&client.AppInitRequest{
+				AppName: appName,
+				Domain:  domainName,
+			})
+		})
+
+		It("app init succeeds", func() {
+			Expect(appInitErr).NotTo(HaveOccurred())
+		})
+
+		It("app init creates a copilot directory", func() {
+			Expect("./copilot").Should(BeADirectory())
+		})
+
+		It("app ls includes new application", func() {
+			Eventually(cli.AppList, "30s", "5s").Should(ContainSubstring(appName))
+		})
+
+		It("app show includes domain name", func() {
+			appShowOutput, err := cli.AppShow(appName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(appShowOutput.Name).To(Equal(appName))
+			Expect(appShowOutput.URI).To(Equal(domainName))
+		})
+	})
+
+	Context("when adding new environment", Ordered, func() {
```

Comment:
samesies! And for the other `Context` as well. The `Ordered` deco is applied on all the `It` specs, so if the `It` specs within a `Context` container doesn't need to be ordered, we don't need the deco.
```suggestion
	Context("when adding new environment", func() {
```